src/cpp/rtps/writer/StatelessWriter.cpp

```diff
@@ -34,6 +34,7 @@
 #include <vector>
 
 #include <fastrtps/log/Log.h>
+#include "../RTPSDomainImpl.hpp"
```

Comment:
```suggestion
#include "rtps/RTPSDomainImpl.hpp"
```

Reply:
`"rtps/RTPSDomainImpl.hpp"` will not be found using relative paths. The suggestion is wrong.